react-carousel-component/my-app/src/IndexButtons.js

```diff
@@ -0,0 +1,19 @@
+import './IndexButtons.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircle as solid } from '@fortawesome/free-solid-svg-icons';
+import { faCircle as regular } from '@fortawesome/free-regular-svg-icons';
```

Comment:
Interesting that you use FontAwesomeIcon here but react-icons in other places. It would be good to standardize on one or the other.